R/epi.temperature.R

```diff
@@ -1,17 +1,19 @@
-#' @title Get volumetrically averaged epilimnion temp
+#' Get volumetrically averaged epilimnion temp
 #' 
-#' @description Calculates volumetrically weighted average epilimnetic temperature using
-#' the supplied water temperature timeseries. If the lake is not stratified, the bottom 
-#' of the epilimnion is calculated as the full depth of the lake. 
-#' 
-#' @param wtr a numeric vector of water temperature in degrees C.
-#' @param depths a numeric vector corresponding to the depths (in m) of the wtr measurements
-#' @param bthA a numeric vector of cross sectional areas (m^2) corresponding to bthD depths
-#' @param bthD a numeric vector of depths (m) which correspond to areal measures in bthA
+#' Calculates volumetrically weighted average epilimnetic temperature using the
+#' supplied water temperature timeseries. If the lake is not stratified, the
+#' bottom of the epilimnion is calculated as the full depth of the lake.
 #' 
 #' 
-#' @seealso \code{\link{hypo.temperature}}, \code{\link{whole.lake.temperature}}
```

Comment:
Is there a reason to remove the `@seealso` section? If not, please leave them. They are sometimes useful and certainly don't hurt.

Reply:
I think this was just an errant delete so no no reason.